packages/fxa-settings/src/pages/InlineRecoverySetup/container.tsx

```diff
@@ -70,8 +70,12 @@ export const InlineRecoverySetupContainer = ({
   const [oAuthError, setOAuthError] =
     useState<FinishOAuthFlowHandlerResult['error']>();
 
+  // Force a network read so this status reflects the in-flight account,
```

Comment:
nit: network _read_? Comment could be moved above where we set the fetch policy to be more clear but it's fine as-is

Reply:
Fixed